src/core/ext/census/gen/trace_context.pb.c

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2016, Google Inc.
+ * Copyright 2017, Google Inc.
```

Comment:
Copyright should (I believe) be for the year the first version was submitted, not modifications. Making it a year range (2016-2017) should also be OK.